src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
+    	String lines = "--------------------------------------------"; 
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?"); 
+        System.out.println(lines);
+
+
+        String line;
+        Task[] tasks = new Task[100]; 
```

Comment:
Maybe you can create a new class called TaskManager which helps you manage tasks and do some operations on tasks. And you can declare the array of Task in this new class. It will make your project more constructured